src/SprykerEco/Zed/Adyen/AdyenConfig.php

```diff
@@ -44,6 +44,11 @@ class AdyenConfig extends AbstractBundleConfig
     protected const ADYEN_NOTIFICATION_SUCCESS_TRUE = 'true';
     protected const ADYEN_NOTIFICATION_SUCCESS_FALSE = 'false';
 
+    protected const ADYEN_PAYMENT_STATUS_REFUSED = 'refused';
```

Comment:
According to their documentation, the result code will start with a capital letter. So they should be:
`Refused`, `Error`, `Cancelled` and `Authorised`.

Reply:
Yes, I saw.
But in our side we change this status to **strtolower**.
See `CreditCardSaver line 39.`